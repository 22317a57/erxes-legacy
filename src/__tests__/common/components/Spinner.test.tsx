import { mount, shallow } from 'enzyme';
import React from 'react';

import Spinner from '../../../modules/common/components/Spinner';

describe('Spinner component', () => {
  const defaultProps = {
    objective: true
  };

  test('renders Spinner successfully', () => {
    shallow(<Spinner {...defaultProps} />);
  });
});
