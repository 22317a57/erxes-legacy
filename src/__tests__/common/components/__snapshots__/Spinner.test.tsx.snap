// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner component snapshot matches 1`] = `
<div
  className="sc-bdVaJa dzYWQC"
>
  <div
    className="sc-bwzfXH lgzOv"
  />
</div>
`;
